grype/version/helper_test.go

```diff
@@ -1,40 +1,43 @@
 package version
 
 import (
-	"errors"
 	"fmt"
+	"github.com/stretchr/testify/assert"
 	"strings"
 	"testing"
 )
 
 type testCase struct {
-	version    string
-	constraint string
-	satisfied  bool
-	createErr  error
-	constErr   error
-	checkErr   error
+	testName       string
```

Comment:
nit: the common go test convention for this field (if included) is `name` which is consistent with many of the tests in our repos.